hail/python/hailtop/utils/utils.py

```diff
@@ -34,6 +34,13 @@
 }"""
 
 
+def unpack_comma_delimited_inputs(inputs):
+    return [s.strip()
+            for steps in inputs
```

Comment:
I know this is a direct copy from what was there, but it sure seems complicated instead of this:

```python3
return [s.strip() for s in inputs.split(',') if s.strip()]
```